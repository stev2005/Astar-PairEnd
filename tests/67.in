AATTTCGAAGTACGACTTACAATAAGTATCATCAGCCCCGTAGAGGCGCGTGGTGGCCTAGTTTTCACGCTCCCGATTGACCGTAAGCCCCTAATAGTCACAGGCAGATAACGTAGTGACTCAAGTAATCAACGGGGCGGCGTTCTGCGGGCTGCCGCCCGGCTATCTATCGGACTTTACCGTCCAGTGAAAGCAGACCCCGTTGAGGATTCCG
15
GTCCCCCCTGCCTTTAACTTATCGTACCGATATAGACTCAGTGCGCCGGACGTGATTGACGTCCATCCTGGCCAGTCTCTTGGGCGCAACCGCG
CGAGCTAGAAAACTCCGATACCGCGTGAAAA
GAATTACACCAGACTGCGTGACTGACGCGGGATGAGTCGATGGTAGGCACAAGTGGCCTTTGTGATCTCTTAGGTGACTAGTCATTTAAGTAAGGCGTATTATCGGTGAGGGGTTCTGCTAATCG
GACGTAGGCACAGCTCTCAGGTATTCTGGCAAGGGCGCAACCAAGTCCAC
GCACACAGTGTTACGCTTCTAGAAATCGATGGAGAAAAGTGCTGGCAGACCAACCACGGCCATCTTCTAAGGCCCAGCGGGTTGCATGGCTAGGCCGTAGTGAAACAGGGCCGATTAGCGAAGGGATATTGGCTGCATATCCATCTGCGTTGTGAGGTGCGCTTACCAACTGGTCTTAACTTTTCTCT
TAAATTACACCCATTACGTCGAAGTTTGATAATAATTCAAGCCGCCGGTCTGCTAATTGTGGTGGACGCCGTGTCTAACACAGTAGTTCGGAAGGTGTCTATGGGTGTTTTCACAATTTACCAGTGCCCGACCGATGGGCCCGCGGCCGAGTGGCCCGTGATACGA
AGTCTCACTGGCATTACCCGGACCACCGTCTTTGAGTATTTGATCTTGACATCGACTCTGGGCCCCTAGTTCCTAGGAATCAGGGGTCTTTCCAGGGCTAAGCCGCTACTTGACAGTAAGTTTAACTGTGGTATCGAATGAGAAACGTCGGAGCAGTTAGCGCGCGACTTTTGATGCCCGA
GGCGAAGAGTGTCTGGACCAATATCCATTAGCGTTGTCGCCACGTTAAAGAACATTCTGAAACGAACTGTCTAGCTCGTCATCCTAAATGATTCGTGTGAGTTGCAGGGCTGCAAAAACATCTGTCCCATCTTACAATTTCGCGTCGTGTTCCTT
AACCTGAG
CT
CGCGATATAGTGGGTTCGTGGATGGTAAATGCCATGTTCTGAGATCTAAGGGGCAAACACAGGGGGACCCCTGTTCAGCAAATGCTGGAGTCCCTTTTCCAGAGCATCGCGTCCCGAAACGTGTCCTAAACAGG
TGGAATGTATCTACTCTTTTAAGAAGGAAATTGGTGCGGGCTCCAAGTTCGTCAACGGGGGCCATCGCTATATAGTAATAGCTTGTCACTTTAATACGCAGACGATCATAACGACATGACCTACAACCGGGAGTGGGATCATTGTAGTGTATGAAGAGAGAGTTCCCTCAAGATATTTGCGGAATCTTTTGTCCGGGAC
ACGACGAACGGTCGAATTTGGCGTGACCAGCCTTCACCCTATGCCGCCCATTGCGAGTCGCGTCCCCTGGGGCATTTAAACTATCGTTGCCTTCACGGAA
GTGGGCCTCGAAGTCAGAGTGGTTGACAACCGATATAGGCAGCGCTGTTCTCTGACGCGGTTATGAGGGATGGAAT
TTTAGATACAGGGACCACTATCTAACGCGGACCTCTAATCACTTCCACGACCCACGGATAGTCTTGGTGCAGGTCTAGAGGCATGCCACAATACGTTAGCCGTTCCGGTGACAAAGGGGTGGTGTCCGGTTTCGTTTCCGACT
