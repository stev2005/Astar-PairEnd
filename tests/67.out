44
13
56
22
93
82
87
74
2
0
66
97
48
35
62
